histoqc/SaveModule.py

```diff
@@ -36,6 +36,39 @@ def saveFinalMask(s, params):
     return
 
 
+def saveAssociatedImage(s, key:str, dim:int):
+    logging.info(f"{s['filename']} - \tsaveMacro")
```

Comment:
is this line strictly correct? this function would seem to be able to save any type of associated image. it is being called from savemacro, thus the actual image pulled is dependent upon the 'key' provided

are there other images that we will want to save, or is it only the macro which is available? if only the macro this may make more sense as a single function

i think there are two options here:
1) merge these two functions
2) saveMacro in this info needs to be - 'save{key}'? or something of the link?